Jenkinsfiles/post_processing_nuc

```diff
@@ -112,12 +127,19 @@ pipeline {
         }
         stage('Create index.html and bublish reports') {
             steps {
-               dir("${params.resultsPath}/"){
-                  sh "python3 ~/Jenkins-agent/workspace/nixOS/post_processing_nuc/indexer/indexer.py /files/images /webify/build_reports ${params.buildID}"
-                  sh "scp -s -i ~/.ssh/sftpid_ed25519 -r ${params.buildID} sftp_user@95.175.105.23:/upload/build_reports/hydra/${params.buildID}"
-                  sh "ssh -i ~/.ssh/trigid_ed25519 script_trigger@95.175.105.23 -- --index build_reports/hydra/${params.buildID}"
-               }
+              dir("${params.resultsPath}/"){
+                 script {
+                     sh "python3 ~/Jenkins-agent/workspace/${level}/post_processing_orin/indexer/indexer.py /files/images/ /webify/build_reports ${params.buildID}"
+                     if(params.development) {
+                          println("Development: ${params.development}")
+                     } else {
+                          println("Development: ${params.development}")
+                          sh "scp -s -i ~/.ssh/sftpid_ed25519 -r ${params.buildID} sftp_user@95.175.105.23:/upload/build_reports/${params.server}/${params.buildID}"
+                          sh "ssh -i ~/.ssh/trigid_ed25519 script_trigger@95.175.105.23 -- --index build_reports/${params.server}/${params.buildID}"

```

Comment:
Internal IPs on the public/general side repo? Not a regression with this patch - just note that don't treat ci-public as ci-private in the future changes.

Reply:
Yep that has been there for a long time don't remember what we agreed with that one